python/py-librosa/Portfile

```diff
@@ -17,36 +17,39 @@ maintainers         nomaintainer
 description         A python package for music and audio analysis.
 long_description    {*}${description}
 
-checksums           rmd160  eb0af487a176a8c7f89d7fb9a58e707ef0e05ab5 \
-                    sha256  4f7421f91a6049483e4cdd567e2c308e4c7673720cd619ae8565e3395b0a6627 \
-                    size    4708361
+checksums           rmd160  25ae847aac6b9effde1b864637bcc86b1be0e564 \
+                    sha256  63e913b0060bb85e8a21ac06a33803d659c1139b2756442ee419012cf6a1df38 \
+                    size    2210477
 
-python.versions     37 38 39
+python.versions     38 39 310 311
 
 if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
     depends_run-append \
                     port:py${python.version}-audioread \
-                    port:py${python.version}-contextlib2 \
                     port:py${python.version}-decorator \
                     port:py${python.version}-joblib \
+                    port:py${python.version}-lazy_loader \
                     port:py${python.version}-matplotlib \
+                    port:py${python.version}-msgpack \
                     port:py${python.version}-numba \
                     port:py${python.version}-numpy \
                     port:py${python.version}-pooch \
-                    port:py${python.version}-resampy \
                     port:py${python.version}-scikit-learn \
                     port:py${python.version}-scipy \
-                    port:py${python.version}-six \
+                    port:py${python.version}-soxr \
                     port:py${python.version}-soundfile \
-                    port:py${python.version}-threadpoolctl
+                    port:py${python.version}-typing_extensions
 
     depends_test-append \
                     port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-cov \
                     port:py${python.version}-pytest-mpl \
-                    port:py${python.version}-samplerate
+                    port:py${python.version}-resampy \
+                    port:py${python.version}-samplerate \
+                    port:py${python.version}-types-decorator
 
     test.run        yes
```

Comment:
all `test.*` stuff below are the defaults already set by the `python` PG , please remove. (Also the `livecheck.type none` below is redundant, please remove).